.coveragerc

```diff
@@ -1209,6 +1209,7 @@ omit =
     homeassistant/components/tradfri/config_flow.py
     homeassistant/components/tradfri/coordinator.py
     homeassistant/components/tradfri/cover.py
+    homeassistant/components/tradfri/diagnostics.py
```

Comment:
Why add to coverage if there are tests?
And on a separate note why is config flow in there?

Reply:
The config flow is ignored by legacy. We should remove it as soon as we have 100 % coverage of that module.

Reply:
```suggestion
```